conference/react-js/src/containers/Talks/index.jsx

```diff
@@ -124,7 +121,7 @@ class Talks extends Component {
 
   handleResetFiltersClick = () => {
     this.setState({
-      filter: { topic: '', status: '' },
+      filter: { conferenceName: '', title: '', status: '' },
```

Comment:
We want use this function to another data (in My talks and Talks components). So number of fields will be different